specification/storage/resource-manager/Microsoft.Storage/2017-06-01/storage.json

```diff
@@ -757,6 +816,47 @@
             "name": "SkuTier",
             "modelAsString": false
           }
+        },
+        "resourceType":{
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of the resource, usually it is 'storageAccounts'."
+        },
+        "kind": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Indicates the type of storage account.",
+          "enum": [
+            "Storage",
+            "BlobStorage"
+          ],
+          "x-ms-enum": {
+            "name": "Kind",
+            "modelAsString": false
+          }
+        },
+        "locations": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The location of the resource. This will be one of the supported and registered Azure Geo Regions (e.g. West US, East US, Southeast Asia, etc.)."
+        },
+        "capabilities": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SKUCapability"
+          },
+          "description": "The capability information in the specified sku, including file encryption, network acls, change notification, etc."
+        },
+        "restrictions": {
```

Comment:
@veronicagg Just FYI, in Python we consider this non breaking (adding an optional parameter at the end). Please take this opportunity to tell in which language it's breaking (I need to improve my Swagger reviewing skill ;))